src/ralph/filters.py

```diff
@@ -1,6 +1,23 @@
 """
 Ralph tracking logs filters.
 """
+from .exceptions import EmptyEventsCollection, EventKeyError
+
+
+def empty(events):
```

Comment:
should be `not_empty` ? The description is the opposite of the function name.

Reply:
This is a silly filter that is no longer required, I'll remove it.